torch/onnx/_internal/fx/dynamo_exporter.py

```diff
@@ -164,10 +164,7 @@ def export(self) -> torch.onnx.ExportOutput:
         # TODO(wechi): There are several symbolic tracing mechanisms to convert
         # nn.Module to FX graph. We should choose the right one after they are
         # matured.
-        # TODO(titaiwang): Set `tracing_mode` according to `self.options.dynamic_shapes`
-        graph_module, graph_guard = torch._dynamo.export(
-            wrapped_model, *args, aten_graph=True, **kwargs
```

Comment:
Curious that did aten_graph works in any bit when decomposition_table is not provided?

Reply:
I suspect this is not even duplicated. It's just to enable symbolic mode.

Reply:
Commented below. It forces the `make_fx` to happen inside `dynamo.export`. And we had no choice because `dynamic=True` must require `tracing_mode='symbolic'`, which requires `aten_graph=True`. Not anymore.